src/lib/docs/Utilities/SigningRequest.md

```diff
@@ -9,20 +9,28 @@ order: 5
 
 # Signing Request
 
-Signing Requests are a data type used in the Antelope ecosystem to facilitate the seamless transfer of transaction details in a compressed string format between applications and signature providers. Designed to encapsulate and transport transaction data, they can be shared through various methods, such as QR codes and hyperlinks.
+Signing Requests are a data type used in the Antelope ecosystem to facilitate the transferring transaction related data between applications for the purposes of generating a signature. This is done using a compressed string format designed to encapsulate and make it easier to transport data, which can be communicated through various methods such as QR codes and hyperlinks.
 
-Two types of signing requests exist:
+The Wharfkit Signing Request library provides a convenient way to interact with this type of data through the `@wharfkit/signing-request` package. The `SigningRequest` class exported by this package offers methods to easily encode, decode, and resolve incomplete transaction data.
```

Comment:
This sentence makes it sound like you can encode and decode incomplete transaction data
```suggestion
The Wharfkit Signing Request library provides a convenient way to interact with this type of data through the `@wharfkit/signing-request` package. The `SigningRequest` class exported by this package offers methods to easily encode or decode transactions, and resolve incomplete transaction data.
```
The current sentence makes it sound like we can easily encode or decode incomplete transaction data instead of signing requests.